fmn/database/model/user.py

```diff
@@ -8,3 +8,25 @@ class User(Base):
 
     id = Column(Integer, primary_key=True, nullable=False)
     name = Column(UnicodeText, nullable=False, unique=True)
+
+    @classmethod
+    def by_name(cls, session, name):  # pragma: no cover
+        return session.query(cls).filter_by(name=name).first()
```

Comment:
Since 1.4, sqlalchemy recommends using the "2.0" style of making queries : https://docs.sqlalchemy.org/en/14/orm/queryguide.html
Also, it will have to be async.